### DISCUSSION TITLE
Quiz questions improvements

### DISCUSSION THREAD

**Discussion by dolfbarr** [greatfrontend/greatfrontend#65] [Ideas/Feature Requests]
- [flashcards](https://en.wikipedia.org/wiki/Flashcard) way
- Assessments for each quiz question/topic
- React quiz questions

**Comment by yangshun** [comment]
Good idea. Would be nice to add quiz/assessments as well

**Comment by yangshun** [comment]
"Test yourself" questions have been added to all questions.
